src/main/java/usecases/CrazyEights.java

```diff
@@ -13,6 +13,12 @@
     private final Output gameOutput;
     private char suitTracker;
 
+    protected static int getMaxPlayers() {
+        return maxPlayers;
+    }
+
+    private static final int maxPlayers = 5;
```

Comment:
Having a method above a variable declaration seems weird, run the code formatter on this file.